src/index.js

```diff
@@ -120,7 +122,11 @@ export default class SignatureCanvas extends Component {
     this._canvas.addEventListener('mousedown', this._handleMouseDown)
     this._canvas.addEventListener('mousemove', this._handleMouseMove)
     document.addEventListener('mouseup', this._handleMouseUp)
-    window.addEventListener('resize', this._resizeCanvas)
+
+    window.addEventListener('resize', () => {
+      if (!this.props.clearOnResize) return
+      this._resizeCanvas()
```

Comment:
I opted for this approach because `_resizeCanvas()` is called (and necessary) when `componentDidMount()` happens, so I couldn't add the check inside `_resizeCanvas()`. Let me know what you think @agilgur5

Reply:
I put this into a separate `_checkClearOnResize` function that's referenced in both places (reference to the function is also needed for `removeEventListener`).

... and just noticed there was a typo I didn't notice on the last code review -- you accidentally changed `removeEventListener` to `addEventListener`. That went into the release so I'll go ahead and hotfix that now woops